source/reference/mongodb-extended-json.txt

```diff
@@ -294,6 +294,30 @@ Undefined Type
 
    The representation for the JavaScript/BSON undefined type.
 
+   You *cannot* use ``undefined`` in query documents.
+   Consider the following document inserted into the ``people`` collection:
+   
+   .. code-block:: sh
+   
+      db.people.insert( { name : Sally, age : undefined } )
```

Comment:
TIL: quotation marks are optional on key names, but not on values. 

I quote everything, normally (keys and values). Will update.